### DISCUSSION TITLE
第六节课作业（17班）

### DISCUSSION THREAD

**Discussion by Jianfeng777** [InternLM/Tutorial#448] [作业]
请上传第六节课的作业到这里

**Comment by l241025097** [comment]
第六节课作业
[06大模型评测OpenCompass.pdf](https://github.com/InternLM/tutorial/files/14001733/06.OpenCompass.pdf)

**Comment by fine-lab** [comment]
https://blog.csdn.net/SmallZCan_t/article/details/135750254?spm=1001.2014.3001.5501

**Comment by zcg-joker** [comment]
[第六节课作业](https://blog.csdn.net/m0_55764641/article/details/135754414)

**Comment by AxrLH** [comment]
作业：
https://blog.csdn.net/Axr_Samuel/article/details/135732202?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135732202%22%2C%22source%22%3A%22Axr_Samuel%22%7D

**Comment by futtttttttttttt** [comment]
https://zhuanlan.zhihu.com/p/679749059
